content/en/network_monitoring/cloud_network_monitoring/network_analytics.md

```diff
@@ -143,22 +143,80 @@ Neutral tags are tags that are not specific to a client or server, and instead a
-| `gateway_type`          | Type of AWS gateway (Internet, NAT, or Transit).                                              |
-| `gateway_region`        | AWS region of the gateway (for example, `us-east-1`).                                                |
-| `gateway_availability-zone` | Availability zone hosting the gateway (for example, `us-east-1a`).                                |
-| `gateway_public_ip`     | Public IP address assigned to the NAT gateway.                                                |
-| `tgw_attachment_id`     | Unique identifier for the AWS Transit Gateway attachment.                                     |
-| `tgw_attachment_type`   | Type of Transit Gateway attachment (for example, VPC, VPN, Direct Connect).                          |
-| `vpc_endpoint_id`       | Unique identifier for the VPC endpoint.                                                       |
+<table>
+<thead>
+<tr>
+<th style="white-space: nowrap; width: 300px; min-width: 300px;">Tag</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><code>gateway_availability-zone</code></td>
+<td>Availability zone hosting the gateway (for example, <code>us-east-1a</code>).</td>
+</tr>
+<tr>
+<td><code>gateway_id</code></td>
+<td>Unique identifier for the AWS gateway resource.</td>
+</tr>
+<tr>
+<td><code>gateway_public_ip</code></td>
+<td>Public IP address assigned to the NAT gateway.</td>
+</tr>
+<tr>
+<td><code>gateway_region</code></td>
+<td>AWS region of the gateway (for example, <code>us-east-1</code>).</td>
+</tr>
+<tr>
+<td><code>gateway_type</code></td>
+<td>Type of AWS gateway (internet, NAT, or Transit).</td>
+</tr>
+<tr>
+<td><code>intra_availability_zone</code></td>
+<td>Indicates whether network flows are within an availability zone (<code>true</code>), cross-AZ (<code>false</code>), or undetermined (<code>unknown</code>). <strong>Note</strong>: Not applicable for Azure.</td>
+</tr>
+<tr>
+<td><code>intra_region</code></td>
+<td>Indicates whether network flows are within a region (<code>true</code>), cross-region (<code>false</code>), or undetermined (<code>unknown</code>).</td>
+</tr>
+<tr>
+<td><code>is_agent_traffic</code></td>
+<td>Indicates if the traffic was generated by the Datadog Agent.</td>
+</tr>
+<tr>
+<td><code>tgw_attachment_id</code></td>
+<td>Unique identifier for the AWS Transit Gateway attachment.</td>
+</tr>
+<tr>
+<td><code>tgw_attachment_type</code></td>
+<td>Type of Transit Gateway attachment (for example, VPC, VPN, Direct Connect).</td>
+</tr>
+<tr>
+<td><code>tls_cipher_insecure</code></td>
+<td>Indicates if the cipher used is considered secure.</td>
+</tr>
+<tr>
+<td><code>tls_cipher_suite</code></td>
+<td>Identifies the TLS cipher suite used (for example, <code>tls_ecdhe_rsa_with_aes_128_gcm_sha256</code>).</td>
+</tr>
+<tr>
+<td><code>tls_client_version</code></td>
+<td>The TLS versions supported by the client (<code>tls_1.2</code> or <code>tls_1.3</code>).</td>
```

Comment:
```suggestion
<td>The TLS version supported by the client (<code>tls_1.2</code> or <code>tls_1.3</code>).</td>
```
Given that it says `or` between the version numbers, I wanted to check if this should be singular?